### ISSUE TITLE
Chests Have Minimal Loot in Them

### ISSUE THREAD

**Issue by AronaxAE** [ForgottenGlory/Living-Skyrim-2#315]
**LS Version**
2.1.0

**Describe the bug**
It seems like most master chests at the end of dungeons don't have nearly as much loot as they should. I am level 40 right now and most end of dungeon chests I find have one or two cards, a low level potion or poison and a really low tier enchanted weapon. You get the odd good thing out of them, but nine times out of then, there is nothing of real value.

**To Reproduce**
Be a high level and find some master chests.

**Expected behavior**
The chests should have a range of items on level with the character.

**Screenshots**
![Capture](https://user-images.githubusercontent.com/19594452/88136330-81d7cf00-cbd8-11ea-884a-4ab74b46951b.PNG)

**Additional context**
Not sure if this is even a bug or if its intended for balance purposes, but I was told I should report it regardless.

**Comment by jokerstyle00** [comment]
Might be just due to the Scarcity mod and its 2x Less patches.

**Comment by AronaxAE** [comment]
> 
> 
> Might be just due to the Scarcity mod and its 2x Less patches.

Well not sure if I should close this bug report in that case. Still not sure its intended to be that low, even with said mods in the list. I will just wait for FG to do his thing.

**Comment by TwistedModding** [comment]
Ok so I just relooked at the readme for the first time in a little while and this is intended. 

"Scarcity makes the loot you find significantly rarer - it will be difficult to find enchanted items or high-level items in chests, on enemies, and with merchants. Don't be surprised to open a chest only to find it has a handful of coins inside. Enemies will, in almost all cases, only have the armor and weapons they're wearing plus perhaps a few coins and maybe one other item. You may also have to travel across various towns to find a merchant that has a particular item in stock. Genesis' loot module helps offset this slightly by adding potions and other items, but at the cost of having more enemies to fight. If a chest has extra enemies near it, it's likely you'll find better loot inside. Honed Metal can also be utilized if you lack the required smithing/enchanting skill and are looking to purchase a specific item - but be prepared to pay a significant cost for it!" - readme

You can go ahead a close this.

**Comment by recklessnl** [comment]
@ForgottenGlory Is this really the experience we want to have though? Shouldn't the game reward you at higher levels with better loot in the master-level chests at higher levels, which are inherently more difficult to clear and thus should be better rewarded ideally? Previous modlists included scarcity as well but still had more varied loot in there with the [More Interesting Loot mod](https://www.nexusmods.com/skyrimspecialedition/mods/24458), which I personally thought was a better balance that is currently present. Curious to hear other people's thoughts on it and especially @ForgottenGlory's.

**Comment by TwistedModding** [comment]
Alduin save to suggest changing, adding, or removing mods.

But to answer your question you are suppose to *earn* the gear you have(this means by leveling up your enchanting and smithing skills to get your high tier armor maxed out), not just have it handed it to you from a dungeon (or even worse a random high tier chest in a dungeon) regardless of what tier chest it is as pretty much anyone could picklock a master lock with level no points put into lockpicking (as i do quite often actually).

**Comment by recklessnl** [comment]
You're missing the point completely. Nobody is arguing for you "to be handed" top tier gear.

The point is that in game design, you generally want to reward high-level players with appropriate rewards when they clear a high level mission / dungeon / level. This is game design 101, and Skyrim generally does this too in Vanilla. The issue is that scarcity's chest rewards are currently disproportionate to player level and difficulty, as the author of this issue is pointing out. As with all modding, this ultimately comes down to personal preference; I personally enjoy having the opportunity to find good and proper loot on higher player levels, as it gives me more of a reason to complete quests and clear out dungeons. Additionally, low-level players will know that the high-level dungeons that they currently are unable to beat, will have higher level gear that they can come back for later. It creates incentive for leveling up and creates incentive to spend time to clear places out.

From a game design perspective, a big and **heavily protected** master chest at the end of a huge and dangerous level should have more than 2 cards, 5 septims and a weak potion in it. 

Again, this is personal preference for how game design should work, so I'll let @ForgottenGlory give his view on this, but I personally agree with OP that this could be improved.

**Comment by ForgottenGlory** [comment]
Ok so here goes:

I see both sides of the argument. On the one hand, it is definitely more rewarding to have bigger chests have more loot in them. On the other, LS2's economy is pretty fragile as far as loot goes. Giving too much could make the economy even more fragile and easier to break. I'm trying to aim for never having too much gold and nothing to spend it on. I'm well aware that this really hasn't been achieved and I do want to improve on it. I don't really know a good solution to this. Scarcity is good in that the Loot module could be disabled and we could only use the Merchants part of it. 

Another option might be to remove Scarcity entirely and utilize Trade & Barter's difficulty settings to offset the increased loot. I haven't tested this yet but would be open to doing so.

Yet another option is to include mods that modify the loot further. I'm a fan of Narrative Loot and Dynamic Dungeon Loot. I have no idea how these will interact with Scarcity until I try them - it may make Scarcity pointless, it may make it even more necessary.

Thinking about it more, the real point of contention here is the inclusion of Scarcity and the decision comes down to keep it or remove it. I don't know what the correct decision is - I'm torn between trying to keep the economy in check and also giving people rewarding loot. 

I don't want a situation where you roll up to Whiterun and drop 10k gold no problem, I want that to be a huge decision point: "I only have 11000 gold, do I _really_ need this armor that costs 10000?" or "YES! I finally have enough to buy this really great weapon I've been saving up for."

I'm open to discussion on how best to make that happen.

**Comment by TwistedModding** [comment]
In that case then dynamic dungeon loot seems like the best choice here. 

If loaded after scarcity I'd imagine scarcity would remove x items from all the chests and enemies etc. While DDL will add loot to chests at the end of the dungeon to make you feel more rewarded. The only issue I can think of with adding DDL is that some of the legendary or "ultra rare" items might sky rocket in price because of supply and demand.

**Comment by KNGLogan** [comment]
well merchants only have like a 1000 gold if it was that rare i would not sell it for that

**Comment by recklessnl** [comment]
First of all, thanks for being open-minded about this and being open to having this discussion @ForgottenGlory.

For what it's worth, anecdotally I can vouch for Dynamic Dungeon Loot adding a **lot** of fun and variety to dungeon crawls in my previous custom modded playthrough, as it made boss-level chests at the end much more unpredictable and thus more fun and rewarding. I will add that I manually tweaked the spawn rates in xedit for my needs and I feel it will definitely be necessary to tweak it. But the randomness and the much more varied loot made that playthrough more fun and rewarding that playing with only scarcity.

I will also add that I feel that high-level players, once they are masters of multiple guilds and are powerful _should_ end up getting rich. It shouldn't be easy, but it should also absolutely be achievable in the mid-late to late game. It's something you worked for to achieve. Additionally, players focusing and investing in Speech should also be rewarded for doing so (by getting better prices, which is already happening in LS2, but just wanted to point this out).

Therefore, imo the general game design should be that the higher level the player is, and the higher level the dungeon / quest is, the higher the rewards generally should be. Dynamic Dungeon Loot can definitely help to achieve this.

**Comment by TwistedModding** [comment]
@KNGLogan In my experience they have upwards of 1500 or even 2000 (from what i've seen sometimes) I know what you're talking about though. 

The thing is Supply and Demand dynamically changes the price of items based on what is available, so say DDL for example spawns in an item thats not even super rare but is above your current level and what you've encountered and whats in stock at the time, the price might be super inflated even though its an item you would sell regardless of the price. 

I get this example from the previous bugs reports with supply and demand over pricing things.

(and btw sorry @recklessnl if I came off as rude to you earlier as that was not my intent, reading it back I kinda sounded snobby)

**Comment by recklessnl** [comment]
@TwistedModding No worries at all! I appreciate that you guys are all open minded about this.

**Comment by KNGLogan** [comment]
yeah my steel arrows are worth over 20,000

**Comment by ForgottenGlory** [comment]
The Supply & Demand issue should hopefully be fixed in 2.2.0, the Extinction setting in the Supply & Demand MCM can be set to around 35% to have a higher chance of prices resetting.

I'm using DDL in Dungeons & Deviousness and there are times you can get really lucky and find something extremely high level for what you currently are - an Ebony Helmet at level 3, for example. That would be my primary concern with DDL, but I agree the variety it provides is fantastic.

**Comment by jokerstyle00** [comment]
I've been playing since 2.0's Beta 6, I wanted to add my input as well.

So my personal take on gold and economy so far is that Scarcity is more annoying than necessary at the moment. There are other ways to "break" the economy so to speak or to have constant reserves of gold; for me, the easiest ways to have a constant cashflow is to invest in Blackthorn and the Carpathian Inn. Even when only partially upgraded, both of those homes/areas give me 500-1000 gold daily. When fully upgraded, I'm raking in thousands a day. Plus with Genesis' increased spawns, there's also the fact that most enemies possess at least one or two items worth a couple hundred gold. 
**TL;DR for this first part: the economy's already easy enough to cheese through enemy loot or through quest homes.**

I don't mind random chests in a dungeon having "sparse" loot so to speak. What frustrates me is when I clear out a long dungeon and the chest at the end is basically empty. It makes clearing dungeons feel unrewarding, and in the case of ones that aren't necessary for major/minor quests such as Bleak Falls, Dead Man's Respite, Ustengrav, etc. the dungeons feel more like busy work than something I want to actually explore and clear, especially if said dungeons aren't linked to unique or powerful rewards such as Dawnbreaker, Spellbreaker, The Pale Blade, etc. 
**TL;DR for this second part: dungeons that aren't directly related to quests or unique equipment feel like a waste of time considering the final chests.**

I think Supply and Demand also needs to have another look at, as there have been multiple reports on seemingly "mundane" items such as ingots or arrows spiking in price to absurd levels. I'm not sure anyone wants to encounter a situation where Steel Arrows, as mentioned above, are worth 20,000 gold per arrow, or Refined Moonstone going for over 60,000 each. My stupid moment was seeing an Empty Common Soul Gem retailing for 6,600 gold each.
**TL;DR for this third part: Supply and Demand does wacky things to the economy and (IMO) is more trouble than it's worth, as it seems to break the economy more than actually managing it.**

I'm not FG, nor do I claim to have anywhere near the modding experience of some of the people in this server. This feedback's based on my own time in Living Skyrim and my past experience as an amateur manual modder. Regardless of how it's viewed, I'd like to thank FG and the team for making a list that's helped made lockdown in San Francisco a lot more tolerable and enjoyable.

**Comment by KNGLogan** [comment]
marring sofia she gives me lots gold like 8 hours and flawless gems

**Comment by daffodilian** [comment]
So this is something I've thought about a lot since early beta of LS2. IMO the economy sucks right now. If you want to make a lot of money really quickly, you can go kill some low level bandits, or maybe rob a poor NPCs dresser drawers. Much quicker and more lucrative than spelunking through the ancient burial tomb that no one has been able to survive for how ever long.

Personally I'd like to see something more closely related to how Skyrim is described:

Bandits having believable loot. Common weapons and armors, food and booze. Lot's of jewelry and gold, spread through their hideouts, because they're bandits and most likely stole them from some passerby on the road and haven't been able to sell them off yet. But the chances of finding something "rare" or extremely expensive would be much much lower.

Draugr ruins really could just use a buff to the 'boss' chests, as most of the loot you find scattered around them is believable.

(i honestly haven't even touched a dwemer ruin in LS2 so i can't form an opinion on them at this point)

Utilizing trade and barter to help make the economy in skyrim feel as harsh as its said to be, in regards to merchants. 
Whiterun would always have the fairest deals, because it's the trade capitol of Skyrim, and you could argue that the merchants would have to be fairer because there could be competition.

Solitude could be more expensive, with finer thing. it's a wealthy city, and most of its citizens can afford the premium costs.

Riften would have merchants trying to rip you off with every transaction, after all they are hurting from being scalped by the thieves guild.

And so on. It would be a bit of work to get to something close to that though, a lot of prices would need patching, leveled lists would need to be examined pretty meticulously. But I think dropping that module of scarcity would be a nice step towards that.

**Comment by recklessnl** [comment]
I think we're all in agreement here that the looting system in dungeons should be tweaked, and that scarcity is the main culprit. I also think that most agree that it's only logical to become rich in the game after doing a lot of quests and clearing out dungeons and slaying tons of enemies - this is just the natural progression of most games in general and it's **not a bad thing at all**. I think @jokerstyle00 makes a good point - it's the nature of the game that you will end up having a lot of gold after a while, and scarcity limiting the boss-level chest loot is doing more harm than good in making the dungeons less exciting and too predictable.

What I think will also help to alleviate the currently too predictable / boring loot in these dungeons issue is adding the [Temper Enchant Service](https://www.nexusmods.com/skyrimspecialedition/mods/21016). Not for the tempering and enchanting services it provides (as Honed Metal already does this, and this is fully compatible with that) but for the addition of a chance of tempered and enchanted gear to be worn by NPCs in these dungeons - _this would help to make it much more rewarding to enter heavily crowded dungeons at higher levels_ - it wouldn't fix the boss-level chest issue right now, but it would certainly help incentivize going into high level dungeons. The way it handles the chances is also _very_ clever and doesn't require any bashed patch for leveled lists - and it natively supports **all** enchantments added by mods too, which can result in some pretty epic battles and less predictable loot.

And in my opinion this would also make these dungeon crawls just more exciting in general - you have a high-level area like Labyrinthian, filled with high level enemies, and there's a chance some of them might have really nice tempered gear or enchanted gear, instead of every single enemy having the exact same predictable loot. The randomness and the chance of getting high gear, even if it's a very small percentage, is a very effective game theory strategy. Hell, this is the whole reason gambling is so popular - there is a chance you might just get lucky and get a great piece of gear in a high level dungeon, and not just from a boss chest, but from the high-level enemies, and that alone makes it worth the time. People in Vegas spend days on slot machines just for this "potential reward". It's a proven dopamine-releasing phenomenon. 

I'd rather have better money sinks for late-game (like the Dragon Hoard perk from Ordinator, exchanging 50k for a perk point, which I think is brilliant game design) then artificially trying to limit an exciting part of any modern game, which is finding great random loot.

I think @jokerstyle00 really made a good point that reinforces this point - why would anybody currently go through unmarked caves and dungeons when all the enemies have the exact same predictable loot (with no chance of quality tempered / enchanted items) and the boss chests will be mostly useless as well? There's very little incentive currently. Adding the chances of higher quality loot will incentivize players to go in and find random treasure, OR come up empty - but the possibility of finding treasure is what makes exploring much more exciting and rewarding.

**Comment by jchritton** [comment]
(JaxomofRuatha on Discord by the way) Pretty much agree entirely with @jokerstyle00 as far as the feel of things, with the addition of it feeling really bad when random enemies either drop really good gear or have it lying around, in comparison to having a huge chest at the very end of a dungeon drop a few Septims and a card. It would be nice if there was a more even distribution in general, but obviously with all the different mods balancing around each other, that's an incredibly difficult ask.

I also agree with the description of the feeling @ForgottenGlory and @daffodilian are describing as the goal, but it is hard to say whether something like Dynamic Dungeon Loot would help with the interaction.

**Comment by ForgottenGlory** [comment]
Reopening this while I consider what to do. Technically not a bug, but will consider as I can.

**Comment by LivingSkyrimFanBoy** [comment]
I hope this discussion is still open to new ideas.

So from my understanding, we need to find a balance between too much and too less loot. Well, I think I may have some mods worth considering, which could help fix this problem. However, I don't have a save, but in order for me help come up with a solution, I'll need to bring up some mods that were never heard of before.

https://www.nexusmods.com/skyrimspecialedition/mods/26

The mod author said that this mod was designed to work well with Scarcity, as it improves the loot while retaining the core features of Scarcity. For more details, please the description of the mod, it explains it better, I believe it may have everything we're looking for, and plus I don't want you to read a long essay, bombarded with text.

https://www.nexusmods.com/skyrimspecialedition/mods/27094

Now, I remember talking to @ForgottenGlory about Morrowloot a while back, he said that it didn't compliment or work well with scarcity in the past. However, I was thinking what if we used a lite version of MLU? A standalone module which has been tweaked and customized by foreverpheonix from "The Phoenix Flavor" modlist. The mod features a "Hybrid Loot" system it tries to find a sweetspot for loot in chests etc.

https://www.nexusmods.com/skyrimspecialedition/mods/26575

This mod is a combination of many mods. It builds on top of existing mods from Lock Related Loot, MLU, and more etc. Since we're looking for better quality loot, the mod has made changes where:
> - No more 100% chance for "special items" drop. Removed enchanced items from normal chests; boss chests have 5% and 10% chance for drop enchanced items drop. 
> - Weapons and armors drop is more believable (for example: draugrs chests have bigger chance for ancient items, warlocks chests have bigger chance for magic items, etc.).

I hope this helps, I'd love to see these improvements just as much as you do.
